variables.tf

```diff
@@ -72,3 +72,8 @@ variable "dcos_instance_os" {
   description = "Operating system to use. Instead of using your own AMI you could use a provided OS."
   default     = "centos_7.4"
 }
+
+variable "name_prefix" {
```

Comment:
I do not currently see a name_prefix for the AWS instance module: https://github.com/dcos-terraform/terraform-aws-instance/blob/master/variables.tf

I'm not sure if this will work.

Reply:
I currently don't see a pull request as well for that module

Reply:
think I already merged it into 0.2

Reply:
Sweet! Confirmed, thanks Jan!
https://github.com/dcos-terraform/terraform-aws-instance/blob/release/v0.2/variables.tf#L89-L92